backend/src/v5/processors/teamspaces/projects/containers/containers.js

```diff
@@ -0,0 +1,84 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { getContainerById, getContainers } = require('../../../../models/modelSettings');
+const { getLatestRevision, getRevisionCount, getRevisions, updateRevisionStatus } = require('../../../../models/revisions');
+const { hasProjectAdminPermissions, isTeamspaceAdmin } = require('../../../../utils/permissions/permissions');
+const { UUIDToString } = require('../../../../utils/helper/uuids');
+const { getFavourites } = require('../../../../models/users');
+const { getProjectById } = require('../../../../models/projects');
+
+const Containers = {};
+
+Containers.getContainerList = async (teamspace, project, user) => {
```

Comment:
@ChristosTsiotsias  this merge conflict didn't resolve properly.

containers folder no longer exist. talk to @Charence if you're unsure how to go about this.